examples/template_telechat.jinja

```diff
@@ -0,0 +1,10 @@
+{{ (messages|selectattr('role', 'equalto', 'system')|list|last).content|trim if (messages|selectattr('role', 'equalto', 'system')|list) else '' }}
+
+{%- for message in messages -%}
+    {%- if message['role'] == 'user' -%}
+        {{- '<_user>' + message['content'] +'<_bot>' -}}
+    {%- elif message['role'] == 'assistant' -%}
+        {{- message['content'] + '<_end>' -}}
+    {%- endif -%}
+{%- endfor -%}
+
```

Comment:
Why does this belong in examples? Could this just be present in the model's tokenizer_config.json?

Reply:
Hi, thanks for your response. Because our chat mode is different from the traditional prompt generation method, not including this might result in significant differences between the generated results and the original results.